src/vanclj/members.clj

```diff
@@ -58,4 +58,9 @@
 
    {:name   "David Dossot"
     :github "https://github.com/ddossot"
-    :twitter "https://twitter.com/ddossot"}])
+    :twitter "https://twitter.com/ddossot"}
+
+   {:name "Ryan McCuaig"
+    :avatar "http://2.gravatar.com/avatar/814feeb82ede969c68d62512f3da2aec"
```

Comment:
This is great and it might work, my only ask would be if you could use `https` instead ;)

Reply:
Seemed to work OK on my local check-out. Fixed the https.